package.json

```diff
@@ -16,7 +16,10 @@
   "author": "Wild Code School",
   "license": "ISC",
   "devDependencies": {
+    "autoprefixer": "^10.4.12",
     "concurrently": "^7.0.0",
-    "husky": "^7.0.0"
+    "husky": "^7.0.0",
+    "postcss": "^8.4.18",
+    "tailwindcss": "^3.2.1"
```

Comment:
tailwind was added in the root package : clean it (and package-lock, and the config files related to tailwind in the root folder : only keep the ones in the frontend folder)